lib/modules/apostrophe-schemas/index.js

```diff
@@ -1330,26 +1330,29 @@ module.exports = {
           return setImmediate(callback);
         },
         form: 'string',
-        addFilter: function(field, cursor) {
-          return cursor.addFilter(field.name, {
-            finalize: function() {
-              var value = cursor.get(field.name);
-              if ((value !== undefined) && (value !== null)) {
-                if (Array.isArray(value) && (value.length === 2)) {
-                  var criteria = {};
-                  criteria[field.name] = { $gte: value[0], $lte: value[1] };
-                } else {
-                  var criteria = {};
-                  criteria[field.name] = self.apos.launder.integer(value);
-                  cursor.and(criteria);
-                }
+      },
+      addFilter: function(field, cursor) {
+        return cursor.addFilter(field.name, {
+          finalize: function() {
+            var value = cursor.get(field.name);
+            if ((value !== undefined) && (value !== null)) {
+              if (Array.isArray(value) && (value.length === 2)) {
+                var criteria = {};
+                criteria[field.name] = { $gte: value[0], $lte: value[1] };
+              } else {
+                var criteria = {};
+                criteria[field.name] = self.apos.launder.integer(value);
+                cursor.and(criteria);
               }
-            },
-            choices: function(callback) {
-              return self.sortedDistinct(field.name, cursor, callback);
             }
-          });
-        }
+          },
+          choices: function(callback) {
+            return self.sortedDistinct(field.name, cursor, callback);
+          },
+          launder: function(s) {
```

Comment:
launder method was also needed for this to work as a manage view or piecesFilter.